CrazyCanvas/Source/ECS/Systems/Player/BenchmarkSystem.cpp

```diff
@@ -60,22 +62,20 @@ void BenchmarkSystem::Tick(LambdaEngine::Timestamp deltaTime)
 	const float32 dt = (float32)deltaTime.AsSeconds();
 
 	ECSCore* pECS = ECSCore::GetInstance();
-	ComponentArray<WeaponComponent>* pWeaponComponents		= pECS->GetComponentArray<WeaponComponent>();
-	ComponentArray<RotationComponent>* pRotationComponents	= pECS->GetComponentArray<RotationComponent>();
+
+	ComponentArray<WeaponComponent>* pWeaponComponents				= pECS->GetComponentArray<WeaponComponent>();
+	const ComponentArray<PositionComponent>* pPositionComponents	= pECS->GetComponentArray<PositionComponent>();
+	ComponentArray<RotationComponent>* pRotationComponents			= pECS->GetComponentArray<RotationComponent>();
```

Comment:
Should be `const ComponentArray<..>* pBlablaComponents = pECS->GetConstComponentArray<..>();`?

Reply:
No, since rotation components are written to.